job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -217,6 +217,26 @@ class WebApi(system: ActorSystem,
           }
         }
       } ~
+        //  DELETE /jobs/<jobId>
+        //  Stop the current job. All other jobs submited with this spark context
```

Comment:
yes but, what I'm trying to say is that the documentation says "current job" , there might be several jobs currently working, you just want to cancel one.